src/figvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/ValidateApplicantServiceControllerTest.java

```diff
@@ -0,0 +1,220 @@
+					.thenReturn(validOrdsServiceResp);
+
+			Mockito.when(figservice.validateApplicantServiceOrdsResponse(VALID_APPL_PARTY_ID, INVALID_ORG_PARTY_ID))
+					.thenReturn(inValidOrgIdOrdsServiceResp);
+
+			Mockito.when(figservice.validateApplicantServiceOrdsResponse(INVALID_APPL_PARTY_ID, VALID_ORG_PARTY_ID))
+					.thenReturn(inValidApplIdOrdsServiceResp);
+
+			Mockito.when(
+					figservice.validateApplicantServiceOrdsResponse(EXCEPTION_ORG_PARTY_ID, EXCEPTION_APPL_PARTY_ID))
+					.thenThrow(FigaroValidationServiceException.class);
+
+		} catch (FigaroValidationServiceException ex) {
+			ex.printStackTrace();
+		}
+
+		validateApplicantServiceController = new ValidateApplicantServiceController(figservice);
+
+	}
+
+	/**
+	 * success
+	 */
+	@Test
+	public void ValidateApplicantServiceControllerSuccess() {
+
+		try {
+			validateApplicantServiceResponse = validateApplicantServiceController
+					.validateApplicantService(VALID_ORG_PARTY_ID, VALID_APPL_PARTY_ID);
+			Assert.assertEquals(VALID_VALIDATION_RESULT, validateApplicantServiceResponse.getValidationResult());
+			Assert.assertEquals(VALID_RESPONSE_MESSAGE, validateApplicantServiceResponse.getRespMsg());
+			Assert.assertEquals(VALID_RESPONSE_CODE, validateApplicantServiceResponse.getRespCode());
+
+		} catch (FigaroValidationServiceException e) {
+			e.printStackTrace();
+			assert (false);
+		}
+
+	}
+
+	/**
+	 * failure response when Invalid Org_Party_Id is passed as input argument
+	 */
+	@Test
+	public void InvalidOrgPartyIdFail() {
+
+		try {
+			validateApplicantServiceResponse = validateApplicantServiceController
+					.validateApplicantService(INVALID_ORG_PARTY_ID, VALID_APPL_PARTY_ID);
+			Assert.assertEquals(INVALID_VALIDATION_RESULT, validateApplicantServiceResponse.getValidationResult());
+			Assert.assertEquals(INVALID_ORG_PARTY_ID_RESPONSE_MESSAGE, validateApplicantServiceResponse.getRespMsg());
+			Assert.assertEquals(INVALID_ORG_PARTY_ID_RESPONSE_CODE, validateApplicantServiceResponse.getRespCode());
+
+		} catch (FigaroValidationServiceException e) {
+			e.printStackTrace();
+			assert (false);
+		}
+
+	}
+
+	/**
+	 * failure response when Invalid Appl_Party_Id input argument
+	 */
+	@Test
+	public void InvalidApplPartyIdFail() {
+
+		try {
+			validateApplicantServiceResponse = validateApplicantServiceController
+					.validateApplicantService(VALID_ORG_PARTY_ID, INVALID_APPL_PARTY_ID);
+			Assert.assertEquals(INVALID_VALIDATION_RESULT, validateApplicantServiceResponse.getValidationResult());
+			Assert.assertEquals(INVALID_APPL_PARTY_ID_RESPONSE_MESSAGE, validateApplicantServiceResponse.getRespMsg());
+			Assert.assertEquals(INVALID_APPL_PARTY_ID_RESPONSE_CODE, validateApplicantServiceResponse.getRespCode());
+
+		} catch (FigaroValidationServiceException e) {
+			e.printStackTrace();
+			assert (false);
+		}
+
+	}
+
+	/**
+	 * exception test
+	 */
+	@Test
+	public void ValidateFigaroControllerException() {
+
+		try {
```

Comment:
to avoid the try catch you can use `assertThrows()` [https://howtodoinjava.com/junit5/expected-exception-example/](https://howtodoinjava.com/junit5/expected-exception-example/), I think your test will fail in the current state

Reply:
I think i misled the test, your test is totally correct @archanasudha